libs/lib-client/define-params/src/params-schema/define-params.module.ts

```diff
@@ -0,0 +1,8 @@
+import { NgModule } from '@angular/core';
+
+import { DefineParamsService } from './define-params.service';
+
+@NgModule({
+  providers: [DefineParamsService],
+})
```

Comment:
Should this module be called `ResourceInterfaceBuilder`? I'm assuming we're putting everything related to the resource interface (params schema) builder here right? Even components